vaultclient/src/test/java/com/scality/vaultclient/dto/CreateAccountRequestDTOTest.java

```diff
@@ -10,7 +10,7 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 //@SuppressWarnings("ConstantConditions")
-public class CreateAccountRequestDTOTest {
+class CreateAccountRequestDTOTest {
```

Comment:
Is this a best practice, or this is something we needed to do for new dependency versions?

Reply:
Since PMD version 6.35.0 JUnit 5 tests can also be run if they’re package-private. Marking them as such is a good practice to limit their visibility. 
https://pmd.github.io/latest/pmd_rules_java_bestpractices.html#junit5testshouldbepackageprivate